src/aioftp/common.py

```diff
@@ -125,17 +168,21 @@ class AsyncListerMixin:
         >>> results = await Context(...)
     """
 
-    async def _to_list(self):
+    async def _to_list(self) -> list[T]:
         items = []
         async for item in self:
             items.append(item)
         return items
 
-    def __await__(self):
+    def __await__(self) -> Generator[None, None, list[T]]:
         return self._to_list().__await__()
 
+    @abc.abstractmethod
+    def __aiter__(self) -> AsyncIterator[T]:
+        pass
```

Comment:
We don't need `pass` in abstract methods. Common way is to add a docstring instead of abstract method body, since abstract method body will never be evaluated:
```py
@abstractmethod
def foo(self) -> T:
    """
    This method do foo
    """
```